cardano-tracer/src/Cardano/Tracer/Utils.hs

```diff
@@ -32,13 +32,18 @@ module Cardano.Tracer.Utils
   , showT
   ) where
 
+#if MIN_VERSION_base(4,18,0)
+-- Do not know why.
```

Comment:
Do you remember what the error was? Is liftA2 not exported anymore, or does it clash with some liftA2 coming in from somewhere else?

Reply:
Something like that